octobot_trading/exchanges/exchanges.py

```diff
@@ -28,7 +28,11 @@ def __init__(self, exchange_manager, matrix_id):
         self.matrix_id = matrix_id
         self.symbols = exchange_manager.exchange_config.traded_symbol_pairs
         self.symbols_by_crypto_currencies = exchange_manager.exchange_config.traded_cryptocurrencies
-        self.time_frames = exchange_manager.exchange_config.traded_time_frames
+        self.time_frames_without_real_time = [
```

Comment:
Same here with `list(set(a) & set(b))`

Reply:
https://github.com/Drakkar-Software/OctoBot-Benchmarks/pull/4

Reply:
Here it's not exactly the same thing, we want to have a filter of ```exchange_manager.exchange_config.traded_time_frames``` by removing the ones in ```exchange_manager.exchange_config.real_time_time_frames```

Reply:
https://github.com/Drakkar-Software/OctoBot-Benchmarks/pull/6